client/my-sites/site-settings/site-icon-setting/index.jsx

```diff
@@ -105,7 +106,13 @@ class SiteIconSetting extends Component {
 			}
 
 			MediaStore.off( 'change', checkUploadComplete );
-			this.saveSiteIconSetting( siteId, media );
+
+			// Check whether upload was successful
+			if ( media ) {
+				this.saveSiteIconSetting( siteId, media );
+			} else {
+				this.props.errorNotice( translate( 'An error occurred while uploading the file' ) );
```

Comment:
Hi! I've found a possible matching string that has already been translated **23** times:
`translate( 'An error occurred while uploading the file.' )` *ES Score: 9.38*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).